### DISCUSSION TITLE
Simplera Medtronic CGM / Companion App

### DISCUSSION THREAD

**Discussion by ffernandezro** [NightscoutFoundation/xDrip#3267] [Ideas]
Hello All!!

Would it be possible to read out the New Simplera Medtronic CGM using the Companion App function?
I was wearing Guardian G4 and Xdrip+ and it works really fine.

The past Monday, I changed the app from Guardian To Simplera App and the new CGM, and now Xdrip is not able to read the gluclose through the notification bar.

The app looks like Guardian, it´s quite simillar!
![Screenshot](https://github.com/NightscoutFoundation/xDrip/assets/8237706/9d184e6b-bf42-4ff8-839a-0cd304e056a0)

Let me know any question
Thanks in advance!

Regards Paco

**Comment by ManuCarrasco67** [comment]
News about this topic? I'm in the same situation...

**Reply by ExPattinatore** [reply]
I also wear Guardian G4 but I can't connect it. What should I do?
![Screenshot_20240905-150951471](https://github.com/user-attachments/assets/384286c5-4caf-4a71-bcfe-b043cbdd985e)
![Screenshot_20240905-150958002](https://github.com/user-attachments/assets/358e853e-cb21-4ad9-83d6-e4665421bcc6)

**Reply by Navid200** [reply]
This discussion was about Simplera and companion app mode.
It was added a long time ago.

**Comment by Navid200** [comment]
@jamorham

**Comment by jamorham** [comment]
Watch out for the next nightly update to support this

**Comment by Navid200** [comment]
Try the Nightly just released and tell us if it works or not.
https://github.com/NightscoutFoundation/xDrip/releases/tag/2024.03.06

**Reply by ManuCarrasco67** [reply]
Thanks a lot for your support, the new versión of xdrip+ works without problems with New Medtronic Simplera sensor. Thanks, thanks and more thanks

**Comment by ffernandezro** [comment]
Ohh yeahhh!
It works for me too.
Thanks for everyone. 

Only a question, anyone could suggest me  a freedns service?
I'm using freedns.afraid with Google Cloud, and it throws errors with the certificates....

Thanks again!!

**Reply by Navid200** [reply]
Please open a new thread and explain the problem with Google Cloud you are experiencing.
I will be happy to help.

**Reply by ffernandezro** [reply]
👌

**Comment by cristiangomezm85** [comment]
Hi,

My son had guardian 4 sensor, and this weekend we have changed to new simplera.

His mobile is an apple SE with all mobile functions locked (provided by hospital). So his mobile is linked vía bluetooth with sensor simplera and new simplera app.

My wife and I used xdrip previously, but now there is no way to connect xdrip with new simplera device. login with browser to carelink via carelink follower option is not working.

About Companion, we have carelink connect app (follower app) but companion neither works with this kind of app.

Can you please confirm me is not possible currently to read simplera from xdrip from a follower mobile? any alternative?

Thanks so much!

**Reply by Navid200** [reply]
If he is using an iPhone, there isn't much he can do with xDrip.
If he used Android, he could use Companion app mode to get readings into xDrip.

I don't follow why you are mentioning CareLink.  If he is using Simplera, how is CareLink related?

**Reply by cristiangomezm85** [reply]
Thanks for clarification @Navid200 .

About carelink, I can explain.

My son has 6 years old, so he still isn't ready to manage diabetes for himself. However, he always has his iPhone everywhere (even at school).

My wife and I have to monitor his glucose levels. Before simplera, with guardian, we used xdrip using CareLink follower option. In this option, you can login in CareLink and remotely allow xdrip scrapp data from It.

Now, with simplera, we can't read from CareLink web. Medtronic has developed an app called "CareLink Connect" for parents/followers and It seams is the only way to read glucose remotely with simplera. This app fails too much, is difficult to know current glucose levels because of UX, and the sound of alerts are too low...

For that reason, I'm trying to find a way to read remotely simplera data vía xdrip. I supose the only solution is to buy an android phone to my son and install on it simplera + xdrip using the Companion App function you mentioned

**Comment by Navid200** [comment]
>  Medtronic has developed an app called "CareLink Connect" for parents/followers and It seams is the only way to read glucose remotely with simplera. 

Please provide a link to anything stating that CareLink Connect can read remotely with Simplera.

**Comment by cristiangomezm85** [comment]
This is carelink Connect: https://play.google.com/store/apps/details?id=com.medtronic.diabetes.carepartner

This is currently the only way we have to read remotely with simplera.

![Screenshot_2024-04-10-23-35-09-380_com medtronic diabetes carepartner](https://github.com/NightscoutFoundation/xDrip/assets/146593642/4def40da-466c-434d-8d5a-361235b09dc7)

**Reply by Navid200** [reply]
I thought you were suggesting that there was an association between CareLink and Simplera.
CareLink, as far as I know, has nothing to do with Simplera.
Perhaps, you were not suggesting that.  

Yes, you can get an Android phone and then, you will be able to use Companion app mode to get readings from Simplera into xDrip.

This is a group that works on an app similar to xDrip for iOS.  You can ask them if they have any suggestions:
https://www.facebook.com/groups/853994615056838

**Reply by cristiangomezm85** [reply]
Hi @Navid200 ,

Sorry for late answer. I can confirm you that medtronic app "CareLink Connect" (for Android and iOS) can be readings remotely to simplera, login with Carelink credentials.

The problem is that this app is not compatible with all devices... And this app has lack of features comparing with xdrip.

We hope someday someone decompile CareLink Connect app and add this remote simplera connection mode to xdrip, it would be great!

Thanks for info about xdrip for iOS. Unfortunaly, my son's iPhone is full locked by hospital, and we can't install any app on It :(

**Comment by Navid200** [comment]
Is anyone getting readings from Simplera through Companion App mode to xDrip?
Is any of you using the latest Nightly or the current stable release?
